### PR TITLE
[SW-423] Set verify button on seed phrase screen to fixed position

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

File: src/popup/pages/SeedPhraseVerifySettings.vue [hunk 1]
```diff
@@ -1,113 +1,93 @@
...
+            :text="mnemonicShuffled[id]"
+            editable
+            @click.native="selectedWordIds.splice(index, 1)"
+          />
+        </template>
+      </div>
     </div>
-
-    <div class="footer">
+    <ModalFooter>
```
Comment (peronczyk) [inline src/popup/pages/SeedPhraseVerifySettings.vue:52]: This page is not a Modal so this component's name makes no sense here.

File: src/popup/components/SeedPhraseNotification.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/variables';
+@use '../../styles/typography';
+
+.seed-phrase-notification {
+  min-height: 176px;
+  border: 2px solid variables.$color-success-dark;
+  border-radius: 16px;
```
Comment (peronczyk) [inline src/popup/components/SeedPhraseNotification.vue:42]: I suggest to create variable for this as this is the same value as the AccountCard has. I talked with Tsvetan and he told me that this radius is a common thing for large cards.

File: src/popup/components/SeedPhraseNotification.vue [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api/dist/vue-composition-api';
+import CheckCircle from '../../icons/check-circle.svg?vue-component';
+import Alert from '../../icons/alert.svg?vue-component';
```
Comment (peronczyk) [inline src/popup/components/SeedPhraseNotification.vue:22]: I suggest to add `...Icon` to icon components as in Vue 3 there is no way of having one-word component names.

Same for the `CheckCircle` above.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/ModalFooter.vue [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  },
+});
+
+</script>
+
+<style lang="scss" scoped>
+.modal-footer {
+  position: sticky;
+  bottom: 0;
+  margin: auto 0 0 0; // Move the footer to the bottom of the container
```
Comment (konradodo) [inline src/popup/components/ModalFooter.vue:25]: margin auto 0 0

File: src/popup/components/SeedPhraseNotification.vue [hunk 5]
```diff
@@ -0,0 +1,77 @@
+<template>
+  <div :class="['seed-phrase-notification', { error: hasError }]">
+    <div class="icon-wrapper">
+      <Alert v-if="hasError" />
+      <CheckCircle v-else />
+    </div>
+    <div class="text">
+      <div>
+        {{
+          hasError
```
Comment (konradodo) [inline src/popup/components/SeedPhraseNotification.vue:10]: move that to computed -> logic should be in script

File: src/styles/variables.scss [hunk 6]
```diff
@@ -15,6 +15,7 @@ $z-index-modal: $z-index-header + 2;
 $z-index-loader: $z-index-modal - 1; // Modals should always be above the loader
 
 $bg-blur-radius: 4px;
+$card-border-radius: 16px;
```
Comment (peronczyk) [inline src/styles/variables.scss:18]: On line 5 and 6 we have variables related to border radiuses with different naming pattern.

File: src/popup/components/InputSelectAsset.vue [hunk 7]
```diff
@@ -64,7 +64,7 @@ export default {
 
   padding: 2px 12px;
   background-color: rgba(variables.$color-black, 0.3);
-  border-radius: 16px;
+  border-radius: variables.$card-border-radius;
```
Comment (peronczyk) [inline src/popup/components/InputSelectAsset.vue:67]: In my opinion that is an accidental coincidence the value is similar. But the scope is different - this input has nothing to do with the cards.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/SeedPhraseBadge.vue [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+export default defineComponent({
+  components: {
+    BtnPlain,
+    CloseIcon,
+  },
+  props: {
+    selected: Boolean,
+    editable: Boolean,
+    text: { type: String, required: true },
```
Comment (subhod-i) [inline src/popup/components/SeedPhraseBadge.vue:24]: Is `required` needed? Isn't it required by default.
The first two props should be `selected: Boolean?` and `editable:Boolean?`
Language is typescript then we can use `?` feature for specifying the Maybe null type.

File: src/popup/pages/SeedPhraseVerifySettings.vue [hunk 9]
```diff
@@ -1,113 +1,93 @@
...
-        </div>
-      </div>
-    </div>
+        :has-error="hasError"
+      />
+    </FixedScreenFooter>
   </div>
 </template>
 
 <script>
```
Comment (subhod-i) [inline src/popup/pages/SeedPhraseVerifySettings.vue:68]: lang='ts'??

File: src/popup/components/SeedPhraseBadge.vue (hunk 8 above)
Comment (pstachel) [reply src/popup/components/SeedPhraseBadge.vue:24]: Yes, we always need to have the text

**Review by peronczyk — APPROVED** [review]

**Review by subhod-i — APPROVED** [review]

**Pull request merged**